packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5428,17 +5429,28 @@ export default class Meeting extends StatelessWebexPlugin {
           },
         };
 
-        this.cleanupLocalStreams();
-
         this.mediaProperties.setMediaDirection(mediaSettings.mediaDirection);
         this.mediaProperties.unsetRemoteMedia();
 
         // when a move to is intiated by the client , Locus delets the existing media node from the server as soon the DX answers the meeting
-        // once the DX answers we establish connection back the media server with only receiveShare enabled
-        // @ts-ignore - reconnectMedia does not accept any argument
-        await this.reconnectionManager.reconnectMedia(mediaSettings).then(() => {
-          Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.MOVE_TO_SUCCESS);
+        // once the DX answers we close the old connection and create new media server connection with only share enabled
```

Comment:
Is this code only applicable for DX? Shouldn't it be any device that is paired? If that's the case, can we change "DX" to "paired device"?

Reply:
It is applicable for all devices that can be paired. Updated the comment